mappings/net/minecraft/registry/Registry.mapping

```diff
@@ -39,6 +39,8 @@ CLASS net/minecraft/unmapped/C_tqxyjqsk net/minecraft/registry/Registry
 		ARG 1 id
 		ARG 2 entry
 	METHOD m_nqoydeav getEntries ()Ljava/util/Set;
+	METHOD m_nxhthmir getId (Lnet/minecraft/unmapped/C_ncpywfca;)Ljava/util/Optional;
```

Comment:
this doesn't return an identifier, it returns a holder reference
checking the code, I'd simply call this `getHolder`
```suggestion
	METHOD m_nxhthmir getHolder (Lnet/minecraft/unmapped/C_ncpywfca;)Ljava/util/Optional;
```